masonry/src/widget/window_handle.rs

```diff
@@ -0,0 +1,142 @@
+// Copyright 2024 the Xilem Authors and the Druid Authors
+// SPDX-License-Identifier: Apache-2.0
+
+//! A titlebar area widget.
+
+use accesskit::{NodeBuilder, Role};
+use dpi::LogicalPosition;
+use smallvec::{smallvec, SmallVec};
+use tracing::{trace_span, Span};
+use vello::kurbo::Point;
+use vello::Scene;
+
+use crate::event::PointerButton;
+use crate::widget::{WidgetMut, WidgetPod};
+use crate::{
+    AccessCtx, AccessEvent, BoxConstraints, EventCtx, LayoutCtx, LifeCycleCtx, PaintCtx,
+    PointerEvent, Size, StatusChange, TextEvent, Widget, WidgetId,
+};
+
+// TODO: Maybe make this configurable somehow
+const DRAG_THRESHOLD: f64 = 8.0;
+
+/// A titlebar area widget.
+///
+/// An area that can be dragged to move the window.
+pub struct WindowHandle<W: Widget> {
+    child: Option<WidgetPod<W>>,
+    last_pos: Option<LogicalPosition<f64>>,
+}
+
+// --- MARK: BUILDERS ---
+impl<W: Widget> WindowHandle<W> {
+    pub fn new(child: W) -> Self {
+        Self {
+            child: Some(WidgetPod::new(child)),
+            last_pos: None,
+        }
+    }
+
+    #[doc(alias = "null")]
```

Comment:
```suggestion
    #[doc(alias = "null")]
```
I don't think this is helpful

Reply:
Me neither, I just copied it from [SizedBox::empty()](https://github.com/linebender/xilem/blob/2832766eff557f0d739f3fe430c90bf914caeaaf/masonry/src/widget/sized_box.rs#L91) to be consistent.